lib/test/reporters.js

```diff
@@ -22,7 +22,7 @@ ObjectDefineProperties(module.exports, {
     configurable: true,
     enumerable: true,
     get() {
-      spec ??= require('internal/test_runner/reporter/spec');
+      spec ??= (...args) => ReflectConstruct(require('internal/test_runner/reporter/spec'), args);
```

Comment:
If you don't use the `function` keyword, it's not possible to do `new spec`, which would be a breaking change.
```suggestion
      spec ??= function Spec() { return ReflectConstruct(require('internal/test_runner/reporter/spec'), arguments); };
```

Reply:
Yes, this is probably the reason why the build tests are failing (Mac, linux, asan).
@aduh95 Is there any better, less-invasive solution than returning the new instance, like we are doing here with Reflect.construct ?

Reply:
Not sure what you mean by less invasive 🤔

Reply:
With Reflect.construct, there is an extra capability of providing newTarget. 
Just meant that.

Reply:
What should be the guideline for a 2nd commit ?
Should I follow with the same commit head, and empty body; since this commit would be a correction only ?

Reply:
The second commit kind of doesn't matter, all commits will be squashed into the first one upon landing – but if you don't want to think about it, you can use `git commit --fixup HEAD` which will reuse the same commit message with the `fixup!` prefix.